src/components/LinkButton.tsx

```diff
@@ -0,0 +1,7 @@
+export const LinkButton = () => {
```

Comment:
Why does this component exist in this branch? If it has not been developed yet, please remove from commit and start new branch for implementation there